src/compiler/bundle/test/bundle-output.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { mockBuildCtx, mockCompilerCtx, mockValidatedConfig } from '@stencil/core/testing';
+import { OutputOptions } from 'rollup';
+
+import { BuildCtx, CompilerCtx, ValidatedConfig } from '../../../declarations';
+import { BundleOptions } from '../bundle-interface';
+import { getRollupOptions } from '../bundle-output';
+
+describe('bundle output', () => {
+  describe('getRollupOptions', () => {
+    let config: ValidatedConfig;
+    let compilerCtx: CompilerCtx;
+    let buildCtx: BuildCtx;
+    let bundleOptions: BundleOptions;
+
+    beforeEach(() => {
+      config = mockValidatedConfig();
+      compilerCtx = mockCompilerCtx(config);
+      buildCtx = mockBuildCtx(config, compilerCtx);
+      bundleOptions = {
+        id: 'customElements',
+        platform: 'client',
+        inputs: {
+          index: 'core',
+        },
+      };
+    });
+
+    it('should default `inlineDynamicImports` to `false` in the output section', () => {
+      const options = getRollupOptions(config, compilerCtx, buildCtx, bundleOptions);
+
+      expect((options.output as OutputOptions).inlineDynamicImports).toBe(false);
```

Comment:
I really wish there was a way for `expect` assertions to narrow a type for us. `options.output` could feasibly be `undefined`, `OutputOptions`, or `OutputOptions[]`. Could we add a few assertions to make using this assertion a little safer for us?
```
      expect(options.output).toBeDefined();
      expect(Array.isArray(options.output)).toBe(false);
```